eras/conway/impl/src/Cardano/Ledger/Conway/Era.hs

```diff
@@ -170,4 +174,11 @@ type instance EraRule "TICK" ConwayEra = ShelleyTICK ConwayEra
 
 type instance EraRule "POOL" ConwayEra = ShelleyPOOL ConwayEra
 
--- =================================================
+-- | Bootstrap phase
+hardforkConwayBootstrapPhase :: ProtVer -> Bool
+hardforkConwayBootstrapPhase pv = pvMajor pv == natVersion @9
+
+-- | Starting with protocol version 11, we do not allow unelected committee
+-- members to submit votes.
+hardforkConwayDisallowUnelectedCommitteeFromVoting :: ProtVer -> Bool
+hardforkConwayDisallowUnelectedCommitteeFromVoting pv = pvMajor pv > natVersion @10
```

Comment:
We could be more specific with the naming and call out `intraEraHardforks` as such, instead of just naming all of the triggers with a `hardfork` prefix?